src/controllers/article.controller.js

```diff
@@ -1,3 +1,6 @@
+import slug from 'slug';
+import crypto from 'crypto';
+import { isNull } from 'util';
```

Comment:
We can't afford to have util dependencies to check a null value. Please the equivalent of this provided by Javascript.